students/roviram/Lesson04/trigrams.py

```diff
@@ -0,0 +1,103 @@
+#--------------------------------------------------------#
+# Dev: Miguel Rovira-Gonzalez
+# Week 4 Homework Assignment: Trigrams Assignment
+# Changelog: Miguel Rovira-Gonzalez
+#--------------------------------------------------------#
+
+import itertools
+import sys
+import random
+
+# WORDS = "with his head sunk upon his chest and his hands clasped behind him.".split()
+# WORDS = "I wish I may I wish I might".split()
+
+punctuations = "!" + "-" + "," + "." + "(" + ")" + '"' + '?' + ';' + ":" + '\n'
+replacements = " " * 11
+
+
+def punctuation_removal(text_line, punc, repl):
+    """
+    function to remove punctuations for cleaner text transition
+    """
+    intab = punc
+    outtab = repl
+    transtab = text_line.maketrans(intab, outtab)
+    clean_string = text_line.translate(transtab)
+    return clean_string
+
+
+def read_file(filename):
+    """
+    Read the file and turn it into a list of words
+    :param filename:
+    :return: list of words in the file
+    """
+    words = []
+    # Code wasn't working unless I put the full file path in
+    with open(r"C:\Users\roviram\Python_210\SP_2019_210A_classroom\students\roviram\Lesson04\sherlock.txt") as text:
```

Comment:
Be careful with hardcoding your entire file path, it is usually easier to put things in your working directory and run from there